### PR TITLE
 Add load states

### PR DESCRIPTION
**Pull request by lmiller1990**
Not sure if we want one component that unites these two - can be useful to use them separately?

image

image

### CODE REVIEW

**Comment by julianjuko** [comment]
the shape of this skeleton looks fairly specific to Vamp, when cohesive UI is generic

**Comment by julianjuko** [comment]
but I agree with keeping these separate, yes 👍

File: src/Icons/ThinSpinner.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+const ThinSpinner: React.FC = () => {
+  return (
+    // @ts-ignore
```
Comment (julianjuko) [inline src/Icons/ThinSpinner.tsx:5]: what's the issue here?

Comment (lmiller1990) [reply src/Icons/ThinSpinner.tsx:5]: oops, we dont need it. removed!

**Pull request merged**